src/Controller/EnMarche/CommitteeManagerController.php

```diff
@@ -94,15 +96,35 @@ public function addEventAction(Request $request, Committee $committee, GeoCoder
      *
      * @Security("committee.isApproved()")
      */
-    public function listMembersAction(Committee $committee, CommitteeManager $committeeManager): Response
+    public function listMembersAction(Committee $committee, CommitteeMembershipRepository $repository): Response
     {
         return $this->render('committee_manager/list_members.html.twig', [
             'committee' => $committee,
-            'committee_hosts' => $committeeManager->getCommitteeHosts($committee),
-            'members' => $committeeManager->getCommitteeMemberships($committee),
+            'committee_hosts' => $repository->findHostMembers($committee),
+            'members' => $repository->getCommitteeMembershipsPaginator($committee, 1, self::MAX_MEMBER_PER_PAGE),
         ]);
     }
 
+    /**
+     * @Route("/membres/{page}", condition="request.isXmlHttpRequest()", requirements={"page": "\d+"}, name="app_committee_manager_members_segment", methods={"GET"})
+     *
+     * @Security("committee.isApproved()")
+     */
+    public function getMembersSegmentAction(
+        int $page,
+        Committee $committee,
+        CommitteeMembershipRepository $repository
+    ): Response {
+        return $this->render(
+            'committee_manager/_member_rows.html.twig',
+            [
+                'committee' => $committee,
+                'members' => $repository->getCommitteeMembershipsPaginator($committee, $page, self::MAX_MEMBER_PER_PAGE),
+                'is_supervisor' => $this->getUser()->isSupervisor(),
```

Comment:
What about using the user resolver to inject the `UserInterface $user` instead of using the `Controller::getUser` shortcut ?

Reply:
done